### DISCUSSION TITLE
How to validate if RTE is empty?

### DISCUSSION THREAD

**Discussion by rizchaerul** [mantinedev/mantine#1260] [Q&A] [answered]
I want to validate whether user has fill the RTE or not, the problem is, when I clear the input, the value of RTE is not empty it is 

`<p><br></p>`

But I can't match the string either because if user hit H1 for example, the empty input value will be

`<h1><br></h1>`

So, how to tackle this problem? Thank you

**Comment by rtivital** [comment, marked as answer]
`'<h1><br></h1>'.replace(/<(.|\n)*?>/g, '').trim().length === 0`

**Reply by rizchaerul** [reply]
Thank you!
